### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by mohdzakwan99**

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/users.scss:1]: Please delete this file

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("log_in") %>
```
Comment (anlvq-1058) [inline app/views/sessions/new.html.erb:1]:
```suggestion
<% provide :title, "Log in" %>
```
If you don't use i18n

File: test/controllers/users_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/users_controller_test.rb:1]: delete this file, please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by mohdzakwan99** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**